### PR TITLE
Make EKS 1.27 default

### LINKED ISSUE #7088: [Feature] Make 1.27 the default Kubernetes version in eksctl
### Description
We've launched support for `1.27` a while ago. It'll be a good time to make the `1.27` default choice, atm it's `1.25`
The current tests must be updated to use `1.27`. Our tests are using outdated/old versions for eg., `1.23` which needs improvements.

### PR DESCRIPTION
**Pull request by TiberiuGC**
### Description

Closes #7088 

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by Himangini — APPROVED** [review]
👍🏻

**Review by a-hilaly — APPROVED** [review]
👍

File: integration/tests/update/update_cluster_test.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gexec"
+	clusterutils "github.com/weaveworks/eksctl/integration/utilities/cluster"
```
Comment (a-hilaly) [inline integration/tests/update/update_cluster_test.go:19]: Nit: this should be with the internal imports group

Comment (TiberiuGC) [reply integration/tests/update/update_cluster_test.go]: https://github.com/eksctl-io/eksctl/pull/7117/commits/25e93e11a6a40b08f053fc0df882ff81e1338d5f

**Pull request merged**